org.uqbar.project.wollok.lib/src/wollok.wlk

```diff
@@ -689,7 +668,227 @@ package lib {
+	}
+	
+	object NUM_7 inherits Key {		
+		method getKeyCodes() = [14, 151]
+	}
+	
+	object NUM_8 inherits Key {		
+		method getKeyCodes() = [15, 152]
+	}
+	
+	object NUM_9 inherits Key {		
+		method getKeyCodes() = [16, 153]
+	}
+	
+	object A inherits Key {		
+		method getKeyCodes() = [29]
+	}
+	
+	object ALT inherits Key {		
+		method getKeyCodes() = [57, 58]
+	}
+	
+	object B inherits Key {		
+		method getKeyCodes() = [30]
+	}
+  
+	object BACKSPACE inherits Key {		
+		method getKeyCodes() = [67]
+	}
+	
+	object C inherits Key {		
+		method getKeyCodes() = [31]
+	}
+  
+	object CONTROL inherits Key {		
+		method getKeyCodes() = [129, 130]
+	}
+  
+	object D inherits Key {		
+		method getKeyCodes() = [32]
+	}
+	
+	object DEL inherits Key {		
+		method getKeyCodes() = [67]
+	}
+  
+	object CENTER inherits Key {		
+		method getKeyCodes() = [23]
+	}
+	
+	object DOWN inherits Key {		
+		method getKeyCodes() = [20]
+	}
+	
+	object LEFT inherits Key {		
+		method getKeyCodes() = [21]
+	}
+	
+	object RIGHT inherits Key {		
+		method getKeyCodes() = [22]
+	}
+	
+	object UP inherits Key {		
+		method getKeyCodes() = [19]
+	}
+	
+	object E inherits Key {		
+		method getKeyCodes() = [33]
+	}
+	
+	object ENTER inherits Key {		
+		method getKeyCodes() = [66]
+	}
+	
+	object F inherits Key {		
+		method getKeyCodes() = [34]
+	}
+	
+	object G inherits Key {		
+		method getKeyCodes() = [35]
+	}
+	
+	object H inherits Key {		
+		method getKeyCodes() = [36]
+	}
+	
+	object I inherits Key {		
+		method getKeyCodes() = [37]
+	}
+	
+	object J inherits Key {		
+		method getKeyCodes() = [38]
+	}
+	
+	object K inherits Key {		
+		method getKeyCodes() = [39]
+	}
+	
+	object L inherits Key {		
+		method getKeyCodes() = [40]
+	}
+	
+	object M inherits Key {		
+		method getKeyCodes() = [41]
+	}
+	
+	object MINUS inherits Key {		
+		method getKeyCodes() = [69]
+	}
+	
+	object N inherits Key {		
+		method getKeyCodes() = [42]
+	}
+	
+	object O inherits Key {		
+		method getKeyCodes() = [43]
+	}
+	
+	object P inherits Key {		
+		method getKeyCodes() = [44]
+	}
+	
+	object PLUS inherits Key {		
+		method getKeyCodes() = [81]
+	}
+	
+	object Q inherits Key {		
+		method getKeyCodes() = [45]
+	}
+	
+	object R inherits Key {		
+		method getKeyCodes() = [46]
+	}
+	
+	object S inherits Key {		
+		method getKeyCodes() = [47]
+	}
+	
+	object SHIFT inherits Key {		
+		method getKeyCodes() = [59, 60]
+	}
+	
+	object SLASH inherits Key {		
+		method getKeyCodes() = [76]
+	}
+	
+	object SPACE inherits Key {		
+		method getKeyCodes() = [62]
+	}
+	
+	object T inherits Key {		
+		method getKeyCodes() = [48]
+	}
+	
+	object U inherits Key {		
+		method getKeyCodes() = [49]
+	}
+	
+	object V inherits Key {		
+		method getKeyCodes() = [50]
+	}
+	
+	object W inherits Key {		
+		method getKeyCodes() = [51]
+	}
+	
+	object X inherits Key {		
+		method getKeyCodes() = [52]
+	}
+	
+	object Y inherits Key {		
+		method getKeyCodes() = [53]
+	}
+	
+	object Z inherits Key {		
```

Comment:
This can be less verbose right now by calling a superclass constuctor like
this

```xtend
object W inhertis Key(51)
object X inhertis Key(52)
object Y inhertis Key(53)
```

Preethe close to an enum

On Wed, Dec 30, 2015 at 7:17 PM, Franco Leonardo Bulgarelli <
notifications@github.com> wrote:

> In org.uqbar.project.wollok.lib/src/wollok.wlk
> <https://github.com/uqbar-project/wollok/pull/552#discussion_r48635694>:
>
> > +		method getKeyCodes() = [50]
> > +	}
> > +	
> > +	object W inherits Key {		
> > +		method getKeyCodes() = [51]
> > +	}
> > +	
> > +	object X inherits Key {		
> > +		method getKeyCodes() = [52]
> > +	}
> > +	
> > +	object Y inherits Key {		
> > +		method getKeyCodes() = [53]
> > +	}
> > +	
> > +	object Z inherits Key {		
>
> I think some is asking for enum's here
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uqbar-project/wollok/pull/552/files#r48635694>.
>